test/components/entity/list.spec.js

```diff
@@ -298,6 +297,53 @@ describe('EntityList', () => {
       component.find('.wrap-circle').exists().should.be.false();
       component.find('.resolve-button').exists().should.be.false();
     });
+
+    describe('Edit Entity from Resolve Modal', () => {
+      const openUpdateFromResolve = async () => {
+        testData.extendedEntities.createPast(1);
+        testData.extendedEntityVersions.createPast(2, { baseVersion: 1 });
+        const mockHttp = load('/projects/1/entity-lists/trees/entities', { root: false }).complete();
+        const component = await mockHttp;
+        const resolveModal = component.getComponent(EntityResolve);
+        const updateModal = component.getComponent(EntityUpdate);
+
+        await component.find('.entity-metadata-row .resolve-button').trigger('click');
+        resolveModal.props().state.should.be.true();
+
+        await resolveModal.find('.edit-entity').trigger('click');
+        resolveModal.props().state.should.be.false();
+        updateModal.props().state.should.be.true();
+
+        return { mockHttp, resolveModal, updateModal };
+      };
+
+      it('should come back to resolve modal after update', async () => {
```

Comment:
Could you also test that after the update, the resolve modal is passed the updated entity? You could assert that the updated label is shown in the resolve modal after the update.